specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/examples/connectionGatewayInstallationsGet.json

```diff
@@ -0,0 +1,33 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
```

Comment:
Are those Id all dummy Ids? Is there a way to ensure no internal information is leaked through the samples?

Reply:
I attempted to use the same id's across all the examples. I grabbed that id from other already existing examples

Reply:
That approach would be fine yes.